lib/screenplay_web/controllers/auth_controller.ex

```diff
@@ -37,9 +39,19 @@ defmodule ScreenplayWeb.AuthController do
   end
 
   def callback(
-        conn = %{assigns: %{ueberauth_failure: %Ueberauth.Failure{}}},
+        conn = %{assigns: %{ueberauth_failure: %Ueberauth.Failure{errors: errors}}},
         _params
       ) do
+    error_messages =
+      errors
+      |> Enum.flat_map(fn
+        %Ueberauth.Failure.Error{message: message} when is_binary(message) -> [message]
```

Comment:
Looks like the `message` field is _supposed_ to [always be a string](https://github.com/ueberauth/ueberauth/blob/master/lib/ueberauth/failure/error.ex#L11), but I suppose it doesn't hurt to make sure.